webapp/channels/src/root.html

```diff
@@ -60,23 +63,123 @@
     </style>
 </head>
 <body class='font--open_sans enable-animations'>
+    <div id="initialPageLoadingScreen" class="LoadingScreen LoadingScreen--darkMode">
```

Comment:
@M-ZubairAhmed this loading html could alternatively be included via a [custom partial](https://github.com/jantimon/html-webpack-plugin/tree/main/examples/custom-template) `<%= require('html-loader!./initial_loading_screen.html').default %>` if you wanted to keep the source html next to the component

Reply:
I was wondering if that was possible, but I didn't dig into it enough. That looks great, and I wonder if we could use it to also avoid needing to add the loader's CSS to the `CopyWebpackPlugin`'s configuration

Reply:
This is awesome thank you for suggesting @calebroseland i added this